### PR TITLE
Fix Package constructor not accepting relative packageUriRoot.

### PR DESCRIPTION
**Pull request by lrhn**

### CODE REVIEW

File: test/package_config_impl_test.dart [hunk 1]
```diff
@@ -0,0 +1,128 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
```
Comment (jakemac53) [inline test/package_config_impl_test.dart]: nit: 2020

File: test/package_config_impl_test.dart [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  var unique = Object();
+  var root = Uri.file("/tmp/root/");
+
+  group("LanguageVersion", () {
+    test("minimal", () {
+      var version = LanguageVersion(3, 5);
+      expect(version.major, 3);
+      expect(version.minor, 5);
+    });
+    test("negative major", () {
```
Comment (jakemac53) [inline test/package_config_impl_test.dart:20]: nit: be consistent about newlines between tests (a newline between each is generally preferred, but it should be consistent either way)

**Review by jakemac53 — APPROVED** [review]

File: test/package_config_impl_test.dart (hunk 1 above)
Comment (lrhn) [reply test/package_config_impl_test.dart]: Done.

File: test/package_config_impl_test.dart (hunk 2 above)
Comment (lrhn) [reply test/package_config_impl_test.dart:20]: Will try.
VSCode does not help me with its inserted `Run|Debug` lines.

Comment (jakemac53) [reply test/package_config_impl_test.dart:20]: Heh ya I have experienced that as well

**Pull request merged**